io.github.b43NnUNF4vidFYFhpqaLWy2ANawtRbMtUXZY9Pf.jumpdf.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "io.github.b43NnUNF4vidFYFhpqaLWy2ANawtRbMtUXZY9Pf.jumpdf",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "command": "jumpdf",
+    "modules": [
+        {
+            "name": "poppler",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DBUILD_GTK_TESTS=OFF",
+                "-DBUILD_QT5_TESTS=OFF",
+                "-DBUILD_CPP_TESTS=OFF",
+                "-DBUILD_QT6_TESTS=OFF",
+                "-DBUILD_MANUAL_TESTS=OFF",
+                "-DENABLE_BOOST=OFF",
+                "-DENABLE_UTILS=OFF",
+                "-DENABLE_GTK_DOC=OFF",
+                "-DENABLE_QT5=OFF",
+                "-DENABLE_QT6=OFF",
+                "-DENABLE_CPP=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://poppler.freedesktop.org/poppler-24.08.0.tar.xz",
+                    "sha256": "97453fbddf0c9a9eafa0ea45ac710d3d49bcf23a62e864585385d3c0b4403174"
+                }
+            ]
+        },
+        {
+            "name": "jumpdf",
+            "builddir": true,
+            "buildsystem": "meson",
+            "config-opts": [
+                "--buildtype=release"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "tag": "v1.0.0",
+                    "url": "https://github.com/b43NnUNF4vidFYFhpqaLWy2ANawtRbMtUXZY9Pf/jumpdf.git"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=xdg-config/jumpdf/config.toml:ro"
```

Comment:
no. 1. this should NOT be a file. 2. it's not justified. You should use xdg standard directory to save your configuration. at worst, persist.

Reply:
FWIW it's the build failure.